<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComponentStateCheckIntervalsEditDialog</class>
 <widget class="QDialog" name="ComponentStateCheckIntervalsEditDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <item>
     <widget class="QPushButton" name="confirmButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="infoContainer">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>220</width>
     <height>250</height>
    </rect>
   </property>
   <property name="title">
    <string>各部件状态检查周期（单位：秒）</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>主框架部件：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>数据采集与播发部件：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>数据存储部件：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>数据处理部件：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>用户管理部件：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>系统监控与管理部件</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="mainFrameWorkInterval">
      <property name="maximum">
       <number>999999999</number>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="dataReceiveAndBroadcastInterval">
      <property name="maximum">
       <number>999999999</number>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="dataProcessingInterval">
      <property name="maximum">
       <number>999999999</number>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QSpinBox" name="dataStoringInterval">
      <property name="maximum">
       <number>999999999</number>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QSpinBox" name="userManagerInterval">
      <property name="maximum">
       <number>999999999</number>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QSpinBox" name="systemMonitorAndManagerInterval">
      <property name="maximum">
       <number>999999999</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>mainFrameWorkInterval</tabstop>
  <tabstop>dataReceiveAndBroadcastInterval</tabstop>
  <tabstop>dataProcessingInterval</tabstop>
  <tabstop>dataStoringInterval</tabstop>
  <tabstop>userManagerInterval</tabstop>
  <tabstop>systemMonitorAndManagerInterval</tabstop>
  <tabstop>confirmButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
